src/Utilities/Microphysics/Microphysics.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    Microphysics
+
+Microphysics parameterization functions (for example condensation,
+evaporation, autoconversion rates for cloud water and rain water)
+TODO: Snow, ice, and 2-moment formulation
+"""
+module Microphysics
+
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+
+using Printf
+
+# rates of conversion between microphysics categories
+export qv2qli, ql2qr, qr2qv
+
+# rain fall speed
+export terminal_velocity
+
+# conversion from specific humidity to mixing ratio (TODO - temporary)
+export q2r
+
+"""
+    qv2qli(PhasePartition, T, ρ[, timescale=1])
+
+Return the ql and qi source terms due to condesation/evaporation (for ql)
```

Comment:
We had ``q_liq``, ``q_ice`` etc. elsewhere.

Reply:
Could be settle on q_l and q_i?

Reply:
This way it's 3 symbols and not 5

Reply:
Fine with me. Please discuss with @charleskawczynski . He was in favor of more verbosity before, which is why we changed to ``q_ice`` etc. in the thermodynamics.